src/smt/smt_engine.cpp

```diff
@@ -4893,6 +4901,40 @@ Result SmtEngine::checkSatisfiability(const vector<Expr>& exprs,
   }
 }
 
+vector<Expr> SmtEngine::getUnsatAssumptions(void)
+{
+  Trace("smt") << "SMT getUnsatAssumptions()" << endl;
+  SmtScope smts(this);
+  if (!options::unsatAssumptions())
+  {
+    throw ModalException(
+        "Cannot get unsat assumptions when produce-unsat-assumptions option "
+        "is off.");
+  }
+  if (d_status.isNull()
+      || d_status.asSatisfiabilityResult() != Result::UNSAT
+      || d_problemExtended)
+  {
+    throw RecoverableModalException(
+        "Cannot get unsat assumptions unless immediately preceded by "
+        "UNSAT/VALID response.");
+  }
+  finalOptionsAreSet();
+#if 0
+  if (Dump.isOn("benchmark"))
+  {
+    Dump("benchmark") << GetUnsatCoreCommand();
+  }
+#endif
```

Comment:
Remove #if 0 ... #endif since it's not used.

Reply:
Whoops, it should actually be used, I only disabled this temporarily. Fixed.